### DISCUSSION TITLE
Simple React App inside a Nextjs folder/route???

### DISCUSSION THREAD

**Discussion by nikosantis** [vercel/next.js#14832] [Help] [answered]
I know it is possible to do app nesting with nextjs as your documentation says with [multizones](https://nextjs.org/docs/advanced-features/multi-zones), but is it possible to do it with an app with create-react-app or custom webpack react-app?

example:
locahost:3000 (nextjs route "/")
locahost:3000/dashboard (react app without nextjs)
-------
example.com (nextjs route "/")
example.com/login (nextjs route /login use cookie or localstorage for token)
example.com/dashboard (react app without nextjs and render router if have token or goto /login for login)

for example, only need the principal route and some simple routes need nextjs for staticprops and SEO (and marketing), but, the /dashboard not need ssr or staticProps, only react.

it's advisable? 
or ...
example.com (principal route with seo and marketing calls)
and use app.example.com for "dashboard example app"???

I'm thiking for example in Framer:
https://framer.com/ (i think have ssr)
https://framer.com/projects/ (react virtual dom)
https://framer.com/projects/3QqBiMb6EjwuSVF8MNCR (react virtual dom)
(i know packages have nextjs in other subdomain)
https://packages.framer.com/ (nextjs)

it is advisable to use next js for an app that has a lot of interactions (example: construction of large dynamic forms, where there is a lot of handling of react states)????
This is where my doubt comes from, or do I do everything within nextjs and so I don't conflict with the query I'm making?

I hope you can understand my question.
Excuse my poor english 🤪

**Comment by jamesmosier** [comment, marked as answer]
I would recommend using Next.js for every part of your application. It can handle any use case you have. Whether it is a ton of dynamic forms, a dashboard, a blog, an ecommerce site, anything. Next allows you to opt-in to the various features per page. So if you want some pages to be server rendered then it can do that or if you want some to just be client side rendered it can do that too. 

Also, there is no way for Next to do "app nesting" of other frameworks. Next is *the* framework and combining other frameworks with Next will not work.

**Reply by nikosantis** [reply]
Thanks!!!
